app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,338 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
+{
+
+    /**
+     * @var $model
+     */
+    protected $model;
+    /**
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 86</br>Variable "role_id" is not in valid camel caps format